src/common/SPELLS/talents/priest.js

```diff
@@ -30,6 +30,7 @@ export default {
   AUSPICIOUS_SPIRITS_TALENT: { id: 155271, name: "Auspicious Spirits", icon: "ability_priest_auspiciousspirits" },
   SHADOW_WORD_DEATH_TALENT: { id: 32379, name: "Shadow Word: Death", icon: "spell_shadow_demonicfortitude" },
   SHADOW_CRASH_TALENT: { id: 205385, name: "Shadow Crash", icon: "spell_shadow_shadowfury" },
+  SHADOW_CRASH_TALENT_DAMAGE: { id: 205386, name: "Shadow Crash", icon: "spell_shadow_shadowfury" },
```

Comment:
You might want to move this to the regular spells file instead. This file gets auto generated sometimes when there are a bunch of talent changes, which would likely remove that entry.